### DISCUSSION TITLE
Same feature for the height

### DISCUSSION THREAD

**Discussion by ObaidAshiq** [tailwindlabs/tailwindcss#12137] [Ideas]
As we have different configuration for widhth, why not power it up to use the height as well

**Comment by alex-krasikau** [comment]
Hey @ObaidAshiq!

Do you mean values like `sm`, `lg`, `xl` and so on? Height property uses default spacing options that you can extend in your theme. But powering it up with the larger set of values, like width, doesn't make sense as it's very rare when you need to set a static height to element, in the most cases you either want container content to define the height or use aspect-ratio for better responsive option.

If that doesn't work for, you're free to extend [theme configuration](https://tailwindcss.com/docs/height#customizing-your-theme) or use [arbitrary values](https://tailwindcss.com/docs/height#arbitrary-values) for one-time usages.

**Reply by ObaidAshiq** [reply]
Hey @alex-krasikau 
To be honest, the scenarios where u want to use height as the similar to width are pretter rare.(in terms of using it in conjuction with @container ).
What i was thinking:
Suppose u have a chart or an image u want it to take 100% width when the height of the div (@container) is say 20 rem and to take 50% when its 10 rem in terms of height.
As today we use maually resizable divs(react-resize), so i thought giving it a bit more power may sound cool.
But that is another headache for developers😂
